src/executor/JobExecutor.ts

```diff
@@ -22,18 +22,16 @@ export class JobExecutor {
     this.stopped = true;
   }
 
-  async execute(jobEntity: JobEntity, parameters?: JobParameters, force: boolean = false): Promise<JobResult> {
-    if (!force) {
-      const { added, running } = await this.schedulesRepository.addExecution(jobEntity.name, jobEntity.maxRunning);
-      if (!added) {
-        this.logger.debug('maxRunning reached, skip', {
-          name: jobEntity.name,
-          running,
-        });
-        return {
-          status: ExecutionStatus.maxRunningReached,
-        };
-      }
+  async execute(jobEntity: JobEntity, parameters?: JobParameters): Promise<JobResult> {
+    const { added, running } = await this.schedulesRepository.addExecution(jobEntity.name, jobEntity.maxRunning);
+    if (!added) {
+      this.logger.debug('maxRunning reached, skip', {
```

Comment:
The name cannot be wrong in a "normal" situation, since the job is read from the db just before `execute` is called. So yes, the only case that we would really expect to happen is that `maxRunning` is reached.